compute/aws-ec2.js

```diff
@@ -111,10 +116,10 @@ class EC2 {
    */
   monitor(params) {
     return new Promise((resolve, reject) => {
-      this._ec2.monitorInstances(params, function(err, data) {
+      this._ec2.monitorInstances(params, function (err, data) {
         if (err && err.code === "DryRunOperation") {
           params.DryRun = false;
```

Comment:
What if the user provides DryRun as true?

Reply:
When the `DryRun` parameter is set to `true` it will check whether the required permissions for the action are there. If yes, then it will return `DryRunOperation` else it will return `UnauthorizedOperation`. So after checking permissions with `DryRun = true` we have to make it `DryRun = false` to run the action unless it will afain return  `DryRunOperation` @rajikaimal